src/libs/ReportActionsUtils.js

```diff
@@ -76,18 +76,20 @@ function getSortedReportActions(reportActions, shouldSortInDescendingOrder = fal
 }
 
 /**
- * Finds most recent IOU report action number.
+ * Finds most recent IOU request action ID.
  *
  * @param {Array} reportActions
  * @returns {String}
  */
-function getMostRecentIOUReportActionID(reportActions) {
-    const iouActions = _.where(reportActions, {actionName: CONST.REPORT.ACTIONS.TYPE.IOU});
```

Comment:
I looked for a way to use `where` to locate a subobject property but was unable to. So please let me know if there is a better way to do this in JS.